SeaPublicWebsite/Helpers/PropertyDataHelper.cs

```diff
@@ -0,0 +1,200 @@
+﻿using SeaPublicWebsite.Data;
```

Comment:
Think this is the file we were discussing this morning (previously UserDataHelper) - should we remove its use elsewhere in the code (as a separate ticket of course)?

Reply:
It's definitely closely related. Hopefully we can get rid of it.